test/built-ins/Temporal/PlainDateTime/prototype/until/units-changed.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.until
+description: Units may be changed
```

Comment:
I'm not sure I understand this description, what does this test show? Is it a full test of the `largestUnit` option?

Reply:
I think it's just a bunch of basic cases, showing off that `largestUnit` is respected. I can add some more cases to make it a test of the complete range of acceptable units.

Reply:
```suggestion
description: Largest unit is respected
```